HISTORY.rst

```diff
@@ -19,6 +19,7 @@ Bug tracker at https://github.com/giampaolo/psutil/issues
 - #797: [Linux] net_if_stats() may raise OSError for certain NIC cards.
 - #813: Process.as_dict() should ignore extraneous attribute names which gets
   attached to the Process instance.
+- #825: Fix possible double close and use of unopened socket 
```

Comment:
Please specify the platform (`[Linux]`) and the function which was affected (the high level, end-user psutil function, not the C function)